### PR TITLE
Continued integration  

### PR DESCRIPTION
**Pull request by reidbaker**
Removes main activity and replaces it with landing activity
Adds host and fan buttons to landing activity
Links host button to host activity
Add bluetooth permissions to manifest
Fix crash on device that does not have bluetooth such as emulator
Creates Bluetooth utils
Renames tToast and gives us the ability to toast and log

### CODE REVIEW

File: fanClub/res/layout/activity_landing.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    android:layout_height="match_parent"
+    tools:context=".LandingActivity" >
+
+    <TextView
+        android:id="@+id/txt_greeting"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_centerVertical="true"
+        android:text="@string/hello_world" />
```
Comment (bryansills) [inline fanClub/res/layout/activity_landing.xml]: Can we change this to something more relevant?

Comment (dgreenhalgh) [inline fanClub/res/layout/activity_landing.xml]: Change what, the name of LandingActivity or the Hello World string?

Comment (bryansills) [inline fanClub/res/layout/activity_landing.xml]: The Hello World string

On Mon, Feb 4, 2013 at 6:11 PM, David Greenhalgh
notifications@github.comwrote:

> In fanClub/res/layout/activity_landing.xml:
> 
> > @@ -0,0 +1,34 @@
> > +<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
> > -    xmlns:tools="http://schemas.android.com/tools"
> > -    android:layout_width="match_parent"
> > -    android:layout_height="match_parent"
> > -    tools:context=".LandingActivity" >
> >   +
> > -    <TextView
> > -        android:id="@+id/txt_greeting"
> > -        android:layout_width="wrap_content"
> > -        android:layout_height="wrap_content"
> > -        android:layout_centerHorizontal="true"
> > -        android:layout_centerVertical="true"
> > -        android:text="@string/hello_world" />
> 
> Change what, the name of LandingActivity or the Hello World string?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/TheLastCrusade/fanClub/pull/8/files#r2885233.

## 

Bryan Sills
Georgia Tech - Computer Science
GTRI Student Co-Op
Kappa Sigma Fraternity

Comment (reidbaker) [inline fanClub/res/layout/activity_landing.xml]: Yes I will change the text to "Welcome to fanClub". I also want to change the variable name. How does `welcome_text` sound?

File: fanClub/src/com/lastcrusade/fanclub/HostActivity.java [hunk 2]
```diff
@@ -62,7 +66,7 @@ public void onClick(View v) {
 			
 			@Override
 			public void onClick(View v) {
-				//onHelloButtonClicked();
+                Toaster.iToast(HostActivity.this, "Testing stack");
```
Comment (bryansills) [inline fanClub/src/com/lastcrusade/fanclub/HostActivity.java:69]: More nitpicking, can you fix the tabbing here?

**Comment by bryansills** [comment]
After you fix the other things, I will accept the merge request.

I am wary of the tight coupling of the networking and UI code, but we can fix that later.

**Pull request merged**